integration_tests/tests/flowmachine_server_tests/test_server.test_api_spec_of_flowmachine_query_schemas.approved.txt

```diff
@@ -271,6 +275,41 @@
     ],
     "type": "object"
   },
+  "LocationIntroversion": {
+    "properties": {
+      "aggregation_unit": {
+        "enum": [
+          "admin0",
+          "admin1",
+          "admin2",
+          "admin3"
+        ],
+        "type": "string"
+      },
+      "direction": {
+        "enum": [
+          "both",
+          "in",
+          "out"
+        ],
+        "type": "string"
+      },
+      "end_date": {
+        "format": "date",
+        "type": "string"
+      },
+      "start_date": {
+        "format": "date",
+        "type": "string"
+      }
+    },
+    "required": [
+      "aggregation_unit",
```

Comment:
The `"direction"` parameters is missing after `"aggregation_unit"`.

Reply:
Ah, that's because "direction" is an optional parameter so I thought it would not belong in a group named "required". But I can add it if "required" means something slightly different in this context.

Reply:
You are right, good catch! I ran approvaltests to auto-generate the correct spec, and it included `"direction"` here because you had set `required=True` in the definition of `LocationIntroversionSchema`. So the above needs changing, and then the spec needs to be re-generated. It would be helpful if you could set up your system so that you can do this rather than manually editing the file and us having to re-check it every time the tests fail.

Reply:
And yes, just to confirm "direction" should not be included in this list of required parameters.